style-contact-us.css

```diff
@@ -0,0 +1,320 @@
+body {
+    font-family:'Poppins', Arial, sans-serif; 
+    background-color: #0D0D15;
+    color:#F8F7FF;
+}
+
+header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    height: 112px;
+    width: 1728px;
+    padding: 32px;
+    color: #0D0D15;
+  }
+  
+  .binabox {
+    width: 188px;
+    height: 42px;
+  }
+  
+  .navigation-list {
+    display: flex;
+  }
+  
+  .navigation-item {
+    padding: 8px 20px;
+    gap: 8px;
+  }
+  
+  .nav-link {
+    color: #F5F8FB;
+    text-decoration: none;
+    font-family:'Poppins', Arial, sans-serif; 
+    font-weight: 600;
+    font-size: 16px;
+    line-height: 24px;
+    border: none;
+  }
+  
+  .nav-link-home {
+    color: #F5F8FB;
+    text-decoration: none;
+    font-family:'Poppins', Arial, sans-serif; 
+    font-weight: 600;
+    border-bottom: 2px solid #0265DC;
+    gap: 8px;
+    width: 89px;
+    height: 40px;
+    padding: 8px 20px;
+    font-size: 16px;
+    line-height: 24px;
+  }
+  
+  .header-buttons {
+    display: flex;
+    align-items: center;
+    background-color: transparent;
+    text-decoration: none;
+    list-style: none;
+    font-family:'Poppins', Arial, sans-serif; 
+    color:#F8F7FF;
+    border: 0px;  
```

Comment:
@Alinchik1 .header-buttons is the UL tag, so you don't need to reset such properties as background-color, text-decoration, border. You can easily remove it. Mark this question, I'll explain you.